### PR TITLE
Aerogear 2159

### PR DESCRIPTION
**Pull request by philbrookes**
**Describe what this PR does and why we need it**:
Update ClientConfig command to honour a services excluded services

Changes proposed in this pull request
 - Honour a services excluded services list
 - Add unit tests to cover new feature

**Does this PR depend on another PR (Use this to track when PRs should be merged)**
No

**Which issue this PR fixes (This will close that issue when PR gets merged)**
AEROGEAR-2159

### CODE REVIEW

File: pkg/cmd/clientConfig.go [hunk 1]
```diff
@@ -67,10 +71,27 @@ kubectl plugin mobile get clientconfig`,
 			if err != nil {
 				return err
 			}
+
+			client, err := ccc.mobileClient.MobileV1alpha1().MobileClients(ns).Get(clientID, metav1.GetOptions{})
+			if err != nil {
+				return errors.Wrap(err, "failed to get mobile client with clientID "+clientID)
+			}
+
+			fmt.Printf("spec: %+v\n", client.Spec)
```
Comment (maleck13) [inline pkg/cmd/clientConfig.go]: should this be here?

Comment (philbrookes) [reply pkg/cmd/clientConfig.go]: Oops

File: pkg/cmd/clientConfig.go [hunk 2]
```diff
@@ -67,10 +71,22 @@ kubectl plugin mobile get clientconfig`,
...
+				return errors.Wrap(err, "failed to get mobile client with clientID "+clientID)
+			}
+
 			ms := listServices(ns, ccc.k8Client)
 			for _, svc := range ms {
 				var svcConfig *ServiceConfig
 				var err error
+				includedService := true
+				for _, excluded := range client.Spec.ExcludedServices {
+					if excluded == svc.ID {
```
Comment (maleck13) [inline pkg/cmd/clientConfig.go]: maybe worth adding a trim ?

**Comment by philbrookes** [comment]
Merging with bad build as it is related to a problem in the cluster used for the integration tests.

**Pull request merged**